docs/source/whatsnew/1.4.0.txt

```diff
@@ -0,0 +1,1024 @@
+Release 1.4.0
+=============
+
+:Release: 1.4.0
+:Date: TBD
+
+Highlights
+----------
+
+
+Build
+-----
+
+| BLD: Add Python 3.6 to CI/CD `David
```

Comment:
- I think the style from previous releases is `-` instead of `|`.
- Can we add links to the PRs?

Reply:
I changed the bullet style. 

There's no easy way to find/get the links to the PRs - this was all parsed via the git commit log on master.